toolkit/imageconfigs/scripts/setup_cvm_image.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+set -e
+
+EFIDIR="BOOT"
+KERNEL_VERSION=""
+
+get_kernel_version() {
+    kernel_modules_dir="/usr/lib/modules"
+    KERNEL_VERSION="$(ls $kernel_modules_dir)"
+}
+
+# symlink /boot/efi to ../efi
+cp -a /boot/efi/. /efi
+rm -rf /boot/efi
+ln -s ../efi /boot/efi
+
+# switching to systemd-boot
+# copy sd-boot EFI binary over the grub EFI binary
+cp /lib/systemd/boot/efi/systemd-bootx64.efi /efi/EFI/$EFIDIR/grubx64.efi
```

Comment:
if there's a dependency on how things are setup/installed in another packages, let's mention that here.